phpBB/styles/prosilver/theme/cp.css

```diff
@@ -353,13 +353,17 @@ ol.def-rules li {
 	padding: 5px;
 }
 
+.cplist .topictitle {
+	line-height: 30px;
+}
+
 /* Responsive *CP navigation
 ---------------------------------------- */
 
 @media only screen and (max-width: 900px), only screen and (max-device-width: 900px) {
 	.nojs .tabs a span,
 	.nojs .minitabs a span {
-		letter-spacing: -0.5px;
+		letter-spacing: -0.031em;
```

Comment:
letter-spacing and word-spacing are the only property exceptions to not using em as they have to be fractional which is not possible with px. It is also safe to use them for them as you never inherit this its always explicitly set at the child and not the parent